handler.go

```diff
@@ -88,13 +92,24 @@ func (r requestHandler) handleStreamRequest() {
 		r.out.Fatalf("Failed while preparing the request: %v\n", err)
 	}
 
+	nextBodyFn, allRequestsFn := streamRequestSupplier(streamMsgReader)
```

Comment:
Can you add a comment explaining how `streamRequestSupplier` will work with the warmup and benchmark and stdin?